### PR TITLE
Make manifest and root paths configurable

### PR DESCRIPTION
**Pull request by zalox**
Hi,

I would like the opportunity to not use the cache directory for the manifest paths, please consider this change :)

### CODE REVIEW

**Comment by zalox** [comment]
This addresses #46 in some sense to.

**Comment by mariusbalcytis** [comment]
`manifest_file_path` should be configurable for that, I agree. Could you explain if there is a need for other parameters to be configurable in your case? As they are more like bundle internals (temporary files) and should not make such difference under normal circumstances.

**Comment by zalox** [comment]
Hi,
Thank you for your quick reply! I would agree that `manifest_file_path` is the most useful one!

Scenarios where the others would be useful to move: 
* `root_directory` : If you would like to put your package.json in some suitable directory for your build. For instance, you might have two projects in your build with different `package.json`
* `webpack_entry_config_path` If you would like to provide your own configuration path. It might be useful for debugging, testing of new configurations or if you don't want to do your environment configuration in javascript.

The `json` file I'm uncertain of what it does, might it be useful for the dev-server?

File: README.md [hunk 1]
```diff
@@ -208,6 +208,9 @@ maba_webpack:
                                         # for example, set dev_server_public_path and public_path to overwrite
                                             # http://localhost:8080/compiled/ and /compiled/
                                             # see inside your webpack.config.js for more info
+        # set location of cached manifests. Useful for deploy, when you don't want to include your cache directory
```
Comment (mariusbalcytis) [inline README.md]: Just to make clear - this is not preferred way for deployment. Or do you run `maba:webpack:compile` in one server, then warm up the cache in production?
You should also be aware that manifest can be different for each environment (or whether you use webpack-dev-server or just webpack).

Comment (zalox) [inline README.md]: Thank you :) You are right, that's exactly what we do.

Comment (mvrhov) [inline README.md]: @mariusbalcytis: They are not the only ones. This is the same I'd like to do. I don't want node garbage + a zillion unnecessary files on my production servers

**Pull request merged**